main.go

```diff
@@ -1209,41 +1223,40 @@ func distBetweenAngles(endRad, startRad, originalDist float64) float64 {
 	return total
 }
 
+func average(arr [5]float64) float64 {
+	sum := 0.
+	for _, val := range arr {
+		sum += val
+	}
+	return sum / float64(len(arr))
+}
```

Comment:
couldn't find any functions for this that people already made so just did a simple average over the last 5 samples. we can make it more complicated at a later date